.github/workflows/stale.yml

```diff
@@ -8,6 +8,9 @@ jobs:
     steps:
       - uses: actions/stale@v8
         with:
+          # Overall configuration
+          operations-per-run: 100
```

Comment:
This is bumped up from default of 30 ([ref](https://github.com/actions/stale#operations-per-run)) because we run at a weekly cadence, and each operation is a comment/issue marked/etc. so I think we will quickly run out of ops/workflow if we run 1x a week without increasin gthis.